segmenti/segmenti.py

```diff
@@ -203,6 +203,11 @@ def volontari_con_titolo(queryset):
     return volontari(queryset).filter(titoli_personali__confermata=True)
 
 
+def tutti_i_volontari_di_una_regione(queryset):
```

Comment:
Non credo che in questo caso sia necessario creare un nuovo segmento. Altrimenti bisognerebbe creare altri N segmenti se volessimo aggiungere, ad esempio, "Tutti i Presidenti di una Regione", oppure "Tutti i Presidenti di una Provincia".

Piuttosto, aggiungendo un semplicissimo campo booleano al modello `articoli.models.BaseSegmento`, ad esempio,
```python
sedi_sottostanti = models.BooleanField(default=false, db_index=True,
                                       help_text="Se selezionato, il segmento viene applicato utilizzando la sede "
                                                 "selezionata e le sedi sottostanti.")
```

Allora, potremmo modificare `articoli.models.BaseSegmento.get_extra_filters` da:
```python
def get_extra_filters(self):
    # ...
    if self.sede:
        filters['appartenenze__sede'] = self.sede.pk
    # ...
```

In qualcosa come:
```python
# ...
if self.sede:
    if self.sedi_sottostanti:
        filters['appartenenze__sede__in'] = self.sede.espandi(includi_me=True, pubblici=True)
    else:
        filters['appartenenze__sede'] = self.sede.pk
# ...
```

E magicamente tutti i segmenti gia' esistenti sarebbero applicabili anche a tutte le provincie, le regioni, un Comitato, ecc.